app/Http/Controllers/LoginController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Interns2025b\Http\Controllers;
+
+use Illuminate\Auth\AuthManager;
+use Illuminate\Http\JsonResponse;
+use Interns2025b\Http\Requests\LoginRequest;
+use Symfony\Component\HttpFoundation\Response as Status;
+
+class LoginController extends Controller
+{
+    public function login(LoginRequest $loginRequest, AuthManager $auth): JsonResponse
+    {
+        $credentials = $loginRequest->validated();
+
+        if (!$auth->attempt($credentials)) {
+            if ($loginRequest->expectsJson()) {
```

Comment:
If login failed and the request does not expect json, you do not return anything, i.e. the function continues: a user who is not logged in can get a token.